pytest_bdd/steps.py

```diff
@@ -304,16 +304,19 @@ def inject_fixture(request, arg, value):
     fd.cached_result = (value, 0, None)
 
     old_fd = get_request_fixture_defs(request).get(arg)
-    old_value = get_fixture_value_raw(request, arg)
     add_fixturename = arg not in request.fixturenames
 
+    old_value = get_fixture_value_raw(request, arg)  # Compatibility with pytest < 3.3.2
```

Comment:
may be it's just easier to support only >=3.3.2?

Reply:
The plan is to fix the build and later work on some issues in order to do a major release. So that others will be able to work on fixes as well.